src/legacy/ui/public/vis/editors/default/controls/agg_control_react_wrapper.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { AggControlProps } from './agg_control_props';
+
+interface AggControlReactWrapperProps<T> extends AggControlProps<T> {
+  component: React.FunctionComponent<AggControlProps<T>>;
+}
+
+function AggControlReactWrapper({
```

Comment:
I would suggest to call this component as `DefaultEditorAggControlWrapper`

Reply:
What is a benefit of renaming this to `DefaultEditorAggControlWrapper` since it will be removed in the nearest future ?

Reply:
Just to be consistent with `DefaultEditorAggSelect`. But you're right we'll remove it soon. So the renaming is up to you.